spec/system/shared_fee/admin_creates_shared_fee_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+describe 'Admin tries to create a shared fee' do
+  it 'and successfully views form for shared fee issue' do
+    admin = Admin.create!(email: 'admin@email.com', password: '123456')
+
+    login_as admin, scope: :admin
+    visit root_path
+    click_on 'Lançar Conta Compartilhada'
+
+    expect(current_path).to eq new_shared_fee_path
+    expect(page).to have_field 'Condomínio'
+    expect(page).to have_field 'Descrição'
+    expect(page).to have_field 'Data de Emissão'
+    expect(page).to have_field 'Valor Total'
+    expect(page).to have_button 'Registrar'
+  end
```

Comment:
Esse teste é desnecessário, já que o teste seguinte já preenche esses campos. Se esse teste aqui falha, o seguinte vai falhar também sempre.

Essa funcionalidade precisava de mais testes verificando ações fora do caminho de sucesso. Falamos bastante sobre isso no primeiro papo sobre os feedbacks dos projetos individuais.
Precisam ser feitos testes de autorização de usuários, visitas diretas pelas rotas fora do caminho de links, validações de campos, etc.

Faltam também testes unitários de models e de requisição.

Reply:
A gente discutiu e concordamos que esse teste é desnecessário. A gente utilizou ele de início pra codar o formulário e criar o model, mas o teste seguinte já cumpria a função desse pois era um teste pra preenchimento do form, e aí apagamos esse teste antigo.